src/components/Wallet/ConnectWallet.tsx

```diff
@@ -30,12 +32,25 @@ export const ConnectWallet: React.FC<ConnectWalletProps> = ({ className }) => {
           variant={isConnected ? "wallet" : "primary"}
           className={className}
           iconLeft={
-            isConnected && address ? (
+            isConnected && address && !isSmallWindow ? (
               <img src={ensImage} alt={`avatar for ${ensName}`} />
             ) : undefined
           }
+          rounded={isSmallWindow ? "full" : "default"}
```

Comment:
@fran-ink I don't think we can get rid of `useIsUnderWindowBreakpoint({ size: "sm" })` in the ConnectWallet component. 👀 Thanks for the good feedback above, I think the code is already much better.

Reply:
Yeah, this is good. Just Button didn't need it